extension/src/experiments/model/quickPicks.ts

```diff
@@ -38,10 +39,13 @@ const getItem = (experiment: Experiment, firstThreeColumnOrder: string[]) => ({
           ? formatDate(collectedVal)
           : collectedVal?.value || collectedVal
 
-      return `${splitUpName[splitUpName.length - 1]}:${value}`
+      return `${truncateFromLeft(
+        splitUpName[splitUpName.length - 1],
```

Comment:
While this does keep a user seeing all three values, the default way (truncating at the very end) allows a user to read the entire output on hover with a tooltip: 

<img width="702" alt="image" src="https://user-images.githubusercontent.com/43496356/199125655-cfa3f7dd-8e5f-4136-929d-9e755d84b6df.png">

Personally, I prefer the default vscode way, but what does everyone else think?

Reply:
Having all the values in one line is quite not readable :(

Reply:
I think it's better to optimize for more values vs a tooltip. Means that we'll have to cut names. 

Also, is it possible to highlight names vs values in some way? (e.g. make the bold, or different color for values, etc?)

Reply:
An example

<img width="740" alt="Screen Shot 2022-10-31 at 10 39 28 PM" src="https://user-images.githubusercontent.com/98249521/199146794-d8770e5c-5d30-483b-923d-2e6220074cf2.png">

Reply:
> Also, is it possible to highlight names vs values in some way? (e.g. make the bold, or different color for values, etc?)
> Having all the values in one line is quite not readable :(

I don't think it's possible to apply any markdown or highlighting to the details. We also can't add new lines to any of the text.